IPython/terminal/ptutils.py

```diff
@@ -12,8 +12,12 @@
 
 class IPythonPTCompleter(Completer):
     """Adaptor to provide IPython completions to prompt_toolkit"""
-    def __init__(self, ipy_completer):
-        self.ipy_completer = ipy_completer
+    def __init__(self, shell):
```

Comment:
Added now in a fully backward-compatible way, and explicitly declared the module as private APIs.